CHANGELOG.md

```diff
@@ -51,12 +51,18 @@ Lots has changed, but if you're already migrating to Swift 3 then that should no
 
 ### Lifetime Semantics
 
+Prior to RAC 5.0, `Signal`s lived and continued to emit values (and side effects) until they completed. This was very confusing, even for RAC veterans. So [changes have been made](https://github.com/ReactiveCocoa/ReactiveCocoa/pull/2959) to the lifetime semantics. `Signal`s now live and continue to emit events while either (a) they have observers or (b) they are retained. This clears up a number of unexpected cases and makes `Signal`s much less dangerous.
```

Comment:
I'd rather say

> Unretained `Signal`s now automatically dispose of themselves at the time they are not being observed.

because

> `Signal`s now live and continue to emit events while either (a) they have observers or (b) they are retained.

has always been the case.